src/components/StatisticsCard/Wins.js

```diff
@@ -4,7 +4,8 @@ import LineChart from '../Charts/LineChart';
 import BarChart from '../Charts/BarChart';
 
 const Wins = (props) => {
-  const lastSnapshot = props.tankStats.data.snapshots.at(-1);
+  const stats = props.tankStatsCard ? props.tankStatsCard : props.accountStats.data;
```

Comment:
To simplify the code, you can utilize the logical OR (||) operator
```suggestion
  const stats = props.tankStatsCard || props.accountStats?.data;
```